themes/default/content/docs/intro/console/organizations.md

```diff
@@ -35,65 +35,82 @@ To create an organization:
 1. Provide an organization name, and agree to the terms of service and privacy policy.
 1. Select **Start free trial**.
 
-You can create a new Pulumi organization directly from the Pulumi Console.
+## Joining an Organization
+
+To become a member of a Pulumi organization, you must be invited by an existing Pulumi
+organization administrator or submit a request to the administrator for approval.
+In addition, you may also need to be a member of the third-party organization or group backing the Pulumi organization.
+
+For example, to become a member of a Pulumi organization backed by a GitLab Group,
+you must associate a GitLab identity with your Pulumi account, and also
+be a member of that GitLab group.
+
+## Switching Between Organizations
+
+The organization menu displays your individual account and all of the organizations you belong.
+
+To switch to a different organization:
+
+1. Select the organization  menu at the top of the page.
+1. Select your organization name.
+
+## Organization Roles
+
+| Role | Description |
+|--------|--------|
+| Admin | Administrators have full access to the organization including: [Inviting members]({{< relref "/docs/intro/console/organization-roles#organization-membership" >}}), [creating teams]({{< relref "/docs/intro/console/teams#creating-a-team" >}}) and policies, managing [stack permissions]({{< relref "/docs/intro/console/stack-permissions" >}}) and [role-based access control]({{< relref "/docs/intro/console/organization-roles" >}}), adjusting billing information, and controlling the organization settings. |
```

Comment:
Nit: Should "Inviting members" be capitalized? I don't believe you are supposed to capitalize the first word after a colon, but I reserve the right to be embarrassingly wrong here.